explainer.md

```diff
@@ -48,218 +49,298 @@ Provide web apps with efficient access to built-in (software and hardware) media
   - control over buffer behavior
   - spatial and temporal scalability
 - Decoded and encoding images
-- Reencoding multiple input media streams in order to merge many encoded media streams into one encoded media stream.
+- Re-encoding multiple input media streams in order to merge many encoded media streams into one encoded media stream.
 
 
 ## Proposed solutions
 
-We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in particular [TransformStreams](https://streams.spec.whatwg.org/#ts-class)) and integrate with [MediaStreamTracks](https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack).
+The WebCodecs interface is modeled on well known platform and software codec APIs. This affords the full power of the underlying platform codecs and is intuitive for authors with experience in developing media applications.
+
+Core interfaces include
+
+-   **EncodedAudioChunks** and **EncodedVideoChunks** contain codec-specific encoded media bytes.
 
-**EncodedAudioChunk**s and **EncodedVideoChunk**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
+-   **AudioPacket** contains decoded audio data. It will provide an [AudioBuffer](https://webaudio.github.io/web-audio-api/#audiobuffer) for rendering via [AudioWorklet](https://webaudio.github.io/web-audio-api/#audioworklet).
 
-**AudioPacket** may expose an [AudioBuffer](https://webaudio.github.io/web-audio-api/#audiobuffer) for rendering via [AudioWorklet](https://webaudio.github.io/web-audio-api/#audioworklet), while **VideoFrame** may expose an ImageBitmap for efficient rendering to canvas. Alternatively, AudioPackets and VideoFrames may be passed as opaque handles to other APIs (such as to/from MediaStreamTracks).
+-   **VideoFrame** contains decoded video data. It will provide an [ImageBitmap](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#imagebitmap) for manipulating in WebGL, including rendering to Canvas.
```

Comment:
I think this is accurate for the MVP, but we get enough questions that it's worth adding that we hope to support YUV ArrayBuffers eventually.